<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>farmDialog</class>
 <widget class="QDialog" name="farmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>760</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Welcome To The Farm</string>
  </property>
  <widget class="QLabel" name="Rulelabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>621</width>
     <height>350</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>	欢迎来到农场。
农场中有100个猪圈，每个猪圈可容纳10头猪。
请自行决定初始农场中猪的数目。
请注意，黑猪会用尽可能少的猪圈去容纳，
并且猪圈分配后便不能改变其容纳猪的种类</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>600</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/begin/Beginbutton.jpg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/begin/Beginbutton.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>600</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/begin/Beginbutton.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Run_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>372</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="b_label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>黑猪</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="b_spinBox">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="b_Slider">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="sf_label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>小花猪</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sf_spinBox">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sf_Slider">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="bf_label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>大白花猪</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="bf_spinBox">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="bf_Slider">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>670</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加载存档</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>进入游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
